keentools/facebuilder/pinmode.py

```diff
@@ -273,10 +273,18 @@ def invoke(self, context, event):
         if not head or not head.headobj:
             _log.error('FB CANNOT FIND head or headobj')
             return {'CANCELLED'}
+
+        if get_gpu_backend() == 'METAL':
```

Comment:
Check for supported backend instead. AFAIK they wank to add vulkan next and we don't support it either